### DISCUSSION TITLE
暂定 logo 及其源文件（2020-08-19）

### DISCUSSION THREAD

**Discussion by xaoxuu** [volantis-x/hexo-theme-volantis#548] [General]
![Logo-Cover@3x.png](https://i.loli.net/2020/07/22/2OYbPxZjhdFce3L.png)

![Logo.png](https://i.loli.net/2020/07/21/7bEGF83MQXvmN1f.png)

> 源文件：[Volantis-20200723.sketch.zip](https://github.com/volantis-x/forum/files/4985798/Volantis-20200723.sketch.zip)

> 源文件：[Volantis-20200819.sketch.zip](https://github.com/volantis-x/forum/files/5093906/Volantis-20200819.sketch.zip)

**Comment by Reqwey** [comment]
👏
